agent/containers/images/Makefile

```diff
@@ -66,105 +98,92 @@ _WORKLOAD_RPMS = fio uperf
 # Not intended to be overridden with an environment variable.
 _ALL_RPMS = ${_TOOL_RPMS} ${_WORKLOAD_RPMS}
 
+# Compose lists of all conceivable distro-version strings (e.g.,
+# `_ALL_centos_VERSIONS` including "centos-7", "centos-8", "centos-9",
+# `_ALL_fedora_VERSIONS` including "fedora-32", "fedora-33", and "fedora-34",
+# and `_DISTROS` containing the contents of all the lists) which will be used
+# to drive pattern matching on distro-based target rules.
+_DIGITS := 0 1 2 3 4 5 6 7 8 9
+_NUMBERS := $(patsubst 0%,%,$(foreach t,${_DIGITS},$(foreach o,${_DIGITS},${t}${o})))
```

Comment:
Wow. I get it, I think, but that's rather dense.

Reply:
I'm open to suggestions....  🙂

Reply:
That was more an expression of cautious awe than a comment. I think it takes a bit of work to unpack this nested loop (maybe it's more like a "make comprehension"), but I'm not sure the complexity justifies the paragraphs of description that would be needed to break it down in a comment.

Reply:
I changed the inner loop indices from `t` and `o` to `tens` and `ones`...perhaps that'll be enough of a clue.

I did decide that a detailed "how it works" comment would be over-doing it, and so I went with just a "what it's doing" description instead, figuring that, as long as it works, that's the important part.  🦐